backend/src/v1/routes/external-consumer-routes.ts

```diff
@@ -23,8 +45,24 @@ router.get(
       );
       res.status(200).json(results);
     } catch (e) {
-      res.json({error: true, message: e.message });
+      res.json({ error: true, message: e.message });
     }
   }),
 );
+
+router.delete(
+  '/delete-reports',
```

Comment:
incorrect implementation, need to discuss with team and come up with identifiers for api call and design

Reply:
@sukanya-rath following a discussion with the team couple of weeks ago, we agreed that we should follow proper endpoint paths, 
`
1. /api/v1/pay-transparency/reports - GET
2. /api/v1/pay-transparency/reports - DELETE
`
We will need to communicate with Christa to change their code.